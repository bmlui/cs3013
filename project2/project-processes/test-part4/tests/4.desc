Testing output of background jobs 
