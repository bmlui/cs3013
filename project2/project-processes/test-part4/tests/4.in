bash -c $PWD/tests/4.sh &
sleep 3
exit

