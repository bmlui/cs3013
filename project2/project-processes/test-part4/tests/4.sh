#!/bin/bash

sleep 1
echo "Hello from the script!"
