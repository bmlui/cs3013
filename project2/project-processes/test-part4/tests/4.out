test-part4$ bash -c $PWD/tests/4.sh &
[1]
test-part4$ sleep 3
Hello from the script!
[1] Done: bash -c $PWD/tests/4.sh 
test-part4$ exit
